sequencer/finalizer.go

```diff
@@ -421,6 +421,10 @@ func (f *finalizer) handleTransactionError(ctx context.Context, result *state.Pr
 		log.Errorf("ROM out of counters error, marking tx with Hash: %s as INVALID, errorCode: %s", tx.Hash.String(), errorCode.String())
 		start := time.Now()
 		f.worker.DeleteTx(tx.Hash, tx.From)
+		err := f.dbManager.UpdateTxStatus(ctx, tx.Hash, pool.TxStatusFailed)
+		if err != nil {
+			log.Errorf("failed to delete tx from pool, err: %s", err)
```

Comment:
the log message must be "failed to update tx from pool, err:" (not "delete")

Reply:
ops yep forgot to update the log

Reply:
will make it more explicit like that:
`log.Errorf("failed to update status to failed in the pool for tx: %s, err: %s", tx.Hash.String(), err)`